app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -286,24 +304,72 @@ public void failure(RetrofitError error) {
     private void initChapters(@NonNull ArrayList<Chapter> chapters) {
         addChapters(chapters);
 
-        mViewPagerAdapter = new ChapterFragmentPagerAdapter(this,
-                getSupportFragmentManager(), selectedChapterId);
+        mViewPagerAdapter = new ChapterFragmentPagerAdapter(
+                this,
+                getSupportFragmentManager(), selectedChapterId
+        );
         mViewPager.setAdapter(mViewPagerAdapter);
         mViewPager.setOffscreenPageLimit(3);
         updateSelectionfor(selectedChapterId);
 
-        mTabLayout.setupWithViewPager(mViewPager);
+        recordStartPageView();
```

Comment:
Not sure whether this is against best practise as the user did not really chose to view this content, but always gets back to it, e.g. after viewing an event.

However, it is recommended to have the main url gdgroups.org also indexed. Not sure which part of the app would map bets...